.circleci/config.yml

```diff
@@ -66,6 +66,37 @@ jobs:
           name: Check for Updated Package name
           command: node -e "if (require('./package.json').name=='@adobe/helix-library') { process.exit(1) };"
 
+      - run:
+          name: Semantic Orb Release
+          command: |
+              # analyze commits to find out if any orbs need to be updated
+              npm run semantic-release --dry-run --analyze-commits | tee /tmp/commit-analysis.log
```

Comment:
Will this script run concurrently on the same machine and filesystem? If yes, you might want to use temporary files, as shown here: https://www.cyberciti.biz/tips/shell-scripting-bash-how-to-create-empty-temporary-file-quickly.html